js/src/Map.js

```diff
@@ -37,10 +39,10 @@ export class LeafletMapModel extends widgets.DOMWidgetModel {
       _model_module: 'jupyter-leaflet',
       _view_module: 'jupyter-leaflet',
       center: DEFAULT_LOCATION,
```

Comment:
You should probably reflect the `center` change you made Python-side by setting a default value of `null` here